pandas/core/window.py

```diff
@@ -503,6 +503,9 @@ class Window(_Window):
     * ``general_gaussian`` (needs power, width)
     * ``slepian`` (needs width).
 
+    If ``win_type=None`` a boxcar window is used. To learn more about different
```

Comment:
this is not correct, the default is in fact ``win_type=None``, which means your window is specified by the the ``window`` parameters; *only* if you actually specify a ``win_type`` do you get a specific weighted window.

Reply:
Fundamentally doing a rolling window calculation involves choosing a window, so what window does passing `win_type=None` choose? I thought it was a boxcar?

Is it valid to specify both the window parameters and the window type, or can only one be provided?

Reply:
most operations are an u weighted window
only if you specify win_type do you get a special weighted window (which also has only sum/mean) methods (this is not very common usage, not using win_type is by far the most common)

i think the docs in computation are pretty clear in this

certainly could add more / better examples in the doc string

Reply:
Right, so if you don't specify win_type you get an un-weighted window, which is a boxcar window, which is what my addition says.

I disagree that the docs are pretty clear, and I think this helps clear them up. (I also think it doesn't confuse anything, and I'm 99% sure what I've written is correct!)

Reply:
Yes, the current default uniform unweighted window is apparently a boxcar window (which I only learned by looking up what a boxcar window is). 
I think that I am not alone in that, therefore I would change the explanation slightly to first explain what the default window is, and then only to say that this is called a boxcar window (people should certainly not need to follow a link to know what the meaning of the default is IMO)

Reply:
Ah, I think I understand the confusion now - I don't have time now, but will try and revise later.